torchquad/plots/plot_adaptive_grid.py

```diff
@@ -0,0 +1,29 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def plot_grid(grid, dpi=100):
```

Comment:
For consistency, let's have function and filename match
```suggestion
def plot_adaptive_grid(grid, dpi=100):
```